launch/launch/logging/__init__.py

```diff
@@ -326,52 +406,6 @@ def get_output_loggers(process_name, output_config):
     )
 
 
-def get_screen_handler():
-    """
-    Get the one and only screen logging handler.
-
-    See launch_config() documentation for screen logging configuration.
-    """
-    if launch_config.screen_handler is None:
-        stream = codecs.StreamWriter(sys.stdout, errors='replace')
-        stream.encode = lambda msg, errors='replace': (
-            msg.encode(locale.getpreferredencoding(False), errors).decode(
-                locale.getpreferredencoding(False), errors=errors),
-            msg)
-        launch_config.screen_handler = handlers.StreamHandler(stream)
-        launch_config.screen_handler.setFormatter(launch_config.screen_formatter)
-    return launch_config.screen_handler
-
-
-def get_log_file_path(file_name='launch.log'):
-    return os.path.join(launch_config.log_dir, file_name)
-
-
-def get_log_file_handler(file_name='launch.log'):
-    """
-    Get the logging handler to a log file.
-
-    See launch_config() documentation for application wide log file
-    logging configuration.
-
-    :param: file_name of the log file whose handler is to be retrieved.
-    :return: the logging handler associated to the file (always the same
-    once constructed).
-    """
-    if file_name not in launch_config.file_handlers:
-        if launch_config.log_handler_factory is None:
-            if os.name != 'nt':
-                launch_config.log_handler_factory = handlers.WatchedFileHandler
-            else:
-                launch_config.log_handler_factory = handlers.FileHandler
-        file_path = get_log_file_path(file_name)
-        file_handler = launch_config.log_handler_factory(
-            file_path, encoding='utf-8')
-        file_handler.setFormatter(launch_config.file_formatter)
-        launch_config.file_handlers[file_name] = file_handler
-    return launch_config.file_handlers[file_name]
-
-
 def _make_unique_log_dir(*, base_path):
```

Comment:
@clalancette move upwards maybe?

Reply:
Yeah, I wasn't sure where to have this live.  I can move it above the `LaunchConfig` class now.